data/pirate missions smuggling intro.txt

```diff
@@ -0,0 +1,1217 @@
+			choice
+				`	[Tug at your shirt suavely.] "Well, Lieutenant, I believe our business is done here. Now, if you would kindly ask your people to repair the damage they've done to my cargo..."`
+				`	"So... I'm not in trouble?"`
+					goto trouble
+			`	Ironside growls, points at you emphatically, then stops short.`
+			label trouble
+			`	"It was a surefire tip..." He sighs, glances around, and puts his hand on your shoulder apologetically. "I'm sorry, kid. To tell you the truth, I don't even smoke cigars. I just thought it would look good." Ironside sighs and stamps out the end of his cigarette on a cargo crate. "Listen, if you wouldn't mind keeping quiet about this, uh, accident here..." He puts the dormant cigar in your shirt pocket. "...I think that would be best. Wrap it up, boys."`
+			`	The Republic officers glance at each other, then set about replacing the lids and re-securing the cargo. A short time later, the Republic personnel shuffle off.`
+			`	You wait for someone to pick up the baking supplies, but after awhile, you head off and allow the dockmaster to send the crates to unclaimed storage. Perhaps Alexandra will shed some light on the situation... Nevertheless, you are quite relieved to have finished your contractual obligation.`
+
+#* * *  (Mission 5 END)  * * *
+
+
+
+event "Etarip Intro Finale"
+
+mission "Etarip Intro Finale"
+	name "Head to Earth"
+	description "Alexandra has offered you further work with Etarip Enterprises. Head to <destination> if you'd like to explore the offer."
+	landing
+	
+	to offer
+		has "event: Etarip Intro Finale"
+	
+	destination
+		planet Earth
+	
+	on offer
+		conversation
+			`Almost as soon as you land on <planet>, Alexandra opens a comm line. "I forgot to congratulate you on finishing your deliveries! You must be so excited." Somehow, her voice seems a touch more cynical than you're used to.`
+			choice
+				`	"It was a heck of a deal, if you ask me. Too good..."`
+					goto deal
+				`	"I'm just glad it's over. I don't want to be involved in this kind of stuff anymore."`
+					goto glad
+				`	"What the hell did you get me into? My ship was raided by Navy officers!"`
+			
+			`	You hear an outburst of laughter. "Oh, come on, <first>. It's not like they found anything. And you lived to tell the tale, right?"`
+			choice
+				`	"Yes, but that's not the point - why did they come after me in the first place?"`
+				`	"You used me as a decoy for a drug run, didn't you?"`
+					goto decoy
+			`	Alexandra giggles. "You must look suspicious! I guess it's the face."`
+			choice
+				`	"Har-har. You used me as a decoy for some kind of drug run, didn't you?"`
+			label decoy
+			`	There is a short pause before Alexandra continues, her tone only slightly more serious. "Quite frankly, yes. But don't tell me you bought the line about 'ultra-vego baking goods.' I think you're smarter than that."`

```

Comment:
> "...I think you're smarter than that."

All the more reason why "Etarip Enterprises" doesn't fit. :)

Reply:
Haha; I absolutely agree. It started out as a silly idea, and is probably best suited for a much younger age bracket.